.github/workflows/macOS_x86_64.yml

```diff
@@ -0,0 +1,67 @@
+name: macOS x86_64
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
+  pull_request:
+    types: [opened, synchronize]
+    paths-ignore:
+      - '*.md'
+  release:
+    types: [published]
+    paths-ignore:
+      - '*.md'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
I'm wondering if we should use older builds for compat on release

Reply:
That might be a good idea